Docs: Update Document Content Security Policy options (#30413)
grafana/grafana: docs/sources/administration/configuration.md

> Set to `false` to disable the X-XSS-Protection header, which tells browsers to stop pages from loading when they detect reflected cross-site scripting (XSS) attacks. The default value is `false` until the next minor release, `6.3`.

Reviewer: ```suggestion
Set to `false` to disable the X-XSS-Protection header, which tells browsers to stop pages from loading when they detect reflected cross-site scripting (XSS) attacks. The default value is `true`.
```

Author: @oddlittlebird Please fix in another PR, it's unrelated to this one.

### content_security_policy

> Set to `true` to add the Content-Security-Policy header to your requests. CSP allows to control resources the user agent is allowed to load and helps prevent XSS attacks.

Reviewer 2: ```suggestion
Set to `true` to add the Content-Security-Policy header to your requests. CSP allows you to control resources that the user agent can load and helps prevent XSS attacks.
```
